app-dicts/myspell-en/myspell-en-20180901.ebuild

```diff
@@ -0,0 +1,55 @@
+# Copyright 1999-2018 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI="7"
+
+MY_PV="${PV:0:4}-${PV:4:2}.${PV:6:2}"
+
+inherit myspell-r2
+
+DESCRIPTION="English dictionaries for myspell/hunspell"
+HOMEPAGE="https://extensions.libreoffice.org/extensions/english-dictionaries"
+SRC_URI="https://extensions.libreoffice.org/extensions/english-dictionaries/${MY_PV}/@@download/file/dict-en-${PV}.oxt"
+
+LICENSE="GPL-2 LGPL-2.1 Princeton myspell-en_CA-KevinAtkinson"
+SLOT="0"
+KEYWORDS="alpha amd64 arm ~arm64 hppa ia64 ~mips ppc ppc64 sh sparc x86 ~amd64-fbsd ~x86-fbsd ~amd64-linux ~x86-linux ~ppc-macos ~x64-macos ~x86-macos ~sparc-solaris ~x86-solaris"
+IUSE=""
+
+PLOCALES="en-US en-GB en-CA en-ZA en-AU"
+for lang in ${PLOCALES}; do
+	LUSE+=" l10n_${lang}"
+done
+IUSE+=${LUSE}
+unset lang LUSE
+
+src_prepare() {
+
+	if use l10n_en-GB; then
+		MYSPELL_HYPH=( "hyph_en_GB.dic" )
+	fi
+	if use l10n_en-US; then
+		MYSPELL_THES=( 
+			"th_en_US_v2.dat" 
+			"th_en_US_v2.idx"
+		)
+	fi
+
+	for lang in ${PLOCALES}; do	
+		local mylinguas="${lang//-/_}"
+		if use "l10n_${lang}"; then
+			MYSPELL_DICT+=( "${mylinguas}.aff" "${mylinguas}.dic" )
```

Comment:
Is MYSPELL_DICT initialized somewhere?

Reply:
i believe the inherit myspell-r2 does this

Reply:
Beliefs need to be backed by evidence ;-).